platform/ios/export/export.cpp

```diff
@@ -38,6 +38,11 @@ void register_ios_exporter_types() {
 }
 
 void register_ios_exporter() {
+#ifdef MACOS_ENABLED
+	EDITOR_DEF("export/ios/iosdeploy", "");
+	EditorSettings::get_singleton()->add_property_hint(PropertyInfo(Variant::STRING, "export/ios/iosdeploy", PROPERTY_HINT_GLOBAL_FILE, "*"));
```

Comment:
Might be more readable with underscore, and consistent with the dash in the upstream name: https://github.com/ios-control/ios-deploy

This should also ensure it's capitalized as "iOS Deploy" and not "Iosdeploy".
```suggestion
	EDITOR_DEF("export/ios/ios_deploy", "");
	EditorSettings::get_singleton()->add_property_hint(PropertyInfo(Variant::STRING, "export/ios/ios_deploy", PROPERTY_HINT_GLOBAL_FILE, "*"));
```